app/src/main/java/org/mozilla/fenix/compose/tabstray/TabListItem.kt

```diff
@@ -0,0 +1,183 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose.tabstray
```

Comment:
wondering if this should live in /compose/tabstray or /compose/list

Reply:
I think this is very specific to the tabstray